Java/libraries/recognizers-text-number/src/main/java/com/microsoft/recognizers/text/number/french/parsers/FrenchNumberParserConfiguration.java

```diff
@@ -20,7 +20,7 @@ public FrenchNumberParserConfiguration() {
     }
 
     public FrenchNumberParserConfiguration(NumberOptions options) {
-        this(new CultureInfo(Culture.Portuguese), options);
+        this(new CultureInfo(Culture.French), options);
```

Comment:
Despite this not being related to datetimes, I figured this is more accurate?

Reply:
Thanks for the catch.